docs/MFScope.md

```diff
@@ -4,23 +4,32 @@ This feature allows user to share state belonging to same group.
 
 ## User
 
-A userId can contain a user, a group, or both. 
+A userId can contain a user, a group, appId or all. A full "userId" will now contain:
+- A user (required)
+- group (optional)
+- An appid (optional)
+
 The following format will be used to determine these fields from within a userId:
 
-- A user without a group will contain simply the user: "foo"
-- A user with a group will contain both a user and group separated by "~". Ex: "foo~bar" represents user "foo" who is a part of group "bar"
-- A group without a user will contain "~" before the group ID. Ex: "~bar".
-    - This example represents group "bar" only.
-    - A userId containing just "bar" will be interpreted as the user "bar" that is not a part of a group.
+| Full "userId"   | User | Group | AppId   |
+| --------------- | ---- | ----- | ------- |
+| 123             | 123  | N/A   | N/A     |
+| 123~A           | 123  | ~A    | N/A
```

Comment:
Does this need a final pipe character?

Reply:
Weirdly enough no. But it has one now anyway.